src/add-search-spec.js

```diff
@@ -114,34 +125,36 @@ describe('Search Results', () => {
           'search'
         );
 
-        content['content'] = pageContents;
+        content.stache_page_search_data = pageContents;
 
         if (!fs.existsSync(searchDirPath)) {
           fs.mkdirSync(searchDirPath);
         }
         return writeSearchFile(searchDirPath);
       })
       .then(() => done())
-      .catch(error => {
+      .catch((error: any) => {
         console.log('ERROR', error);
         expect(error).toBeNull();
         done();
       });
   });
 });
-`
+`;
 
-function addSearchSpecToProject() {
+// http://web.archive.org/web/20161108071447/http://blog.osteele.com/posts/2007/12/cheap-monads/
+function addSearchSpecToProject(argv, config) {
+  if ((((config || {}).appSettings || {}).stache || {}).search) {
```

Comment:
I see what's going on, but not sure I'm a fan.  Personally I find the following more readable...

```
if (config &&
    config.appSettings &&
    config.appSettings.stache &&
    config.appSettings.stache.search) {

}
```